### ISSUE TITLE
My Colab install was successful:

### ISSUE THREAD

**Issue by luyh** [notadamking/RLTrader#130]
My Colab install was successful:

....
Successfully built optuna ta empyrical quantstats alembic yfinance Mako idna-ssl pyperclip
Installing collected packages: pbr, stevedore, pyperclip, colorama, cmd2, cliff, colorlog, Mako, python-editor, alembic, optuna, ta, statsmodels, empyrical, multidict, yarl, pycares, aiodns, async-timeout, typing-extensions, idna-ssl, aiohttp, asn1crypto, cryptography, ccxt, configparser, yfinance, quantstats, tensorflow-gpu
  Found existing installation: statsmodels 0.10.0
    Uninstalling statsmodels-0.10.0:
      Successfully uninstalled statsmodels-0.10.0
Successfully installed Mako-1.0.14 aiodns-1.1.1 aiohttp-3.5.4 alembic-1.0.11 asn1crypto-0.24.0 async-timeout-3.0.1 ccxt-1.18.988 cliff-2.15.0 cmd2-0.9.15 colorama-0.4.1 colorlog-4.0.2 configparser-3.7.4 cryptography-2.7 empyrical-0.5.0 idna-ssl-1.1.0 multidict-4.5.2 optuna-0.13.0 pbr-5.4.1 pycares-3.0.0 pyperclip-1.7.0 python-editor-1.0.4 quantstats-0.0.17 statsmodels-0.10.0rc2 stevedore-1.30.1 ta-0.4.5 tensorflow-gpu-1.14.0 typing-extensions-3.7.4 yarl-1.1.0 yfinance-0.1.43

then I used the following command to launch "optimize":

!python ./cli.py optimize

the following was the last bit of the error print-out:

.....
.....

--------------------------------------
| approxkl           | 1.2992781e-15 |
| clipfrac           | 0.0           |
| ep_rewmean         | nan           |
| eplenmean          | nan           |
| explained_variance | 0.00413       |
| fps                | 14            |
| nupdates           | 17            |
| policy_entropy     | 0.0051370724  |
| policy_loss        | 7.4659073e-07 |
| serial_timesteps   | 5950          |
| time_elapsed       | 352           |
| total_timesteps    | 5950          |
| value_loss         | 0.00065950956 |
--------------------------------------
--------------------------------------
| approxkl           | 0.0           |
| clipfrac           | 0.0           |
| ep_rewmean         | nan           |
| eplenmean          | nan           |
| explained_variance | -0.00316      |
| fps                | 15            |
| nupdates           | 15            |
| policy_entropy     | 2.776626e-09  |
| policy_loss        | -4.196167e-07 |
| serial_timesteps   | 5625          |
| time_elapsed       | 326           |
| total_timesteps    | 5625          |
| value_loss         | 0.0054322067  |
--------------------------------------
  2019-07-28 00:23:04,076 - __main__ - optimize_params - INFO - Pruning trial for not making any trades: 1
[I 2019-07-28 00:23:04,076] Setting status of trial#552 as TrialState.PRUNED. 
  2019-07-28 00:23:09,920 - __main__ - optimize - INFO - Finished trials: 554
  2019-07-28 00:23:10,169 - __main__ - optimize - INFO - Best trial: -57.909999999999854
  2019-07-28 00:23:10,170 - __main__ - optimize - INFO - Params: 
  2019-07-28 00:23:10,468 - __main__ - optimize - INFO -     cliprange: 0.14951451376750177
  2019-07-28 00:23:10,468 - __main__ - optimize - INFO -     ent_coef: 8.755803215177085e-06
  2019-07-28 00:23:10,469 - __main__ - optimize - INFO -     gamma: 0.9024819584559397
  2019-07-28 00:23:10,469 - __main__ - optimize - INFO -     lam: 0.8627452728582746
  2019-07-28 00:23:10,469 - __main__ - optimize - INFO -     learning_rate: 0.0617342341331879
  2019-07-28 00:23:10,469 - __main__ - optimize - INFO -     n_steps: 241.9399431970005
  2019-07-28 00:23:10,469 - __main__ - optimize - INFO -     noptepochs: 14.955931188308773
[79d169cd1242:09793] *** Process received signal ***
[79d169cd1242:09793] Signal: Segmentation fault (11)
[79d169cd1242:09793] Signal code: Address not mapped (1)
[79d169cd1242:09793] Failing at address: 0x7fc383fe220d
[79d169cd1242:09793] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x12890)[0x7fc386c8c890]
[79d169cd1242:09793] [ 1] /lib/x86_64-linux-gnu/libc.so.6(getenv+0xa5)[0x7fc3868cb785]
[79d169cd1242:09793] [ 2] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(_ZN13TCMallocGuardD1Ev+0x34)[0x7fc387136e44]
[79d169cd1242:09793] [ 3] /lib/x86_64-linux-gnu/libc.so.6(__cxa_finalize+0xf5)[0x7fc3868cc615]
[79d169cd1242:09793] [ 4] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(+0x13cb3)[0x7fc387134cb3]
[79d169cd1242:09793] *** End of error message ***
---------------------------------------
| approxkl           | 0.0            |
| clipfrac           | 0.0            |
| ep_rewmean         | nan            |
| eplenmean          | nan            |
| explained_variance | 0.0134         |
| fps                | 19             |
| nupdates           | 16             |
| policy_entropy     | 3.3155612e-09  |
| policy_loss        | -6.1670937e-07 |
| serial_timesteps   | 6000           |
| time_elapsed       | 350            |
| total_timesteps    | 6000           |
| value_loss         | 0.00019280183  |
---------------------------------------
  2019-07-28 00:23:28,090 - __main__ - optimize_params - INFO - Pruning trial for not making any trades: 1
[I 2019-07-28 00:23:28,090] Setting status of trial#553 as TrialState.PRUNED. 
  2019-07-28 00:23:31,960 - __main__ - optimize - INFO - Finished trials: 554
  2019-07-28 00:23:32,134 - __main__ - optimize - INFO - Best trial: -57.909999999999854
  2019-07-28 00:23:32,134 - __main__ - optimize - INFO - Params: 
  2019-07-28 00:23:32,308 - __main__ - optimize - INFO -     cliprange: 0.14951451376750177
  2019-07-28 00:23:32,308 - __main__ - optimize - INFO -     ent_coef: 8.755803215177085e-06
  2019-07-28 00:23:32,308 - __main__ - optimize - INFO -     gamma: 0.9024819584559397
  2019-07-28 00:23:32,308 - __main__ - optimize - INFO -     lam: 0.8627452728582746
  2019-07-28 00:23:32,308 - __main__ - optimize - INFO -     learning_rate: 0.0617342341331879
  2019-07-28 00:23:32,308 - __main__ - optimize - INFO -     n_steps: 241.9399431970005
  2019-07-28 00:23:32,308 - __main__ - optimize - INFO -     noptepochs: 14.955931188308773
[79d169cd1242:09792] *** Process received signal ***
[79d169cd1242:09792] Signal: Segmentation fault (11)
[79d169cd1242:09792] Signal code: Address not mapped (1)
[79d169cd1242:09792] Failing at address: 0x7ff93680320d
[79d169cd1242:09792] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x12890)[0x7ff9394ad890]
[79d169cd1242:09792] [ 1] /lib/x86_64-linux-gnu/libc.so.6(getenv+0xa5)[0x7ff9390ec785]
[79d169cd1242:09792] [ 2] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(_ZN13TCMallocGuardD1Ev+0x34)[0x7ff939957e44]
[79d169cd1242:09792] [ 3] /lib/x86_64-linux-gnu/libc.so.6(__cxa_finalize+0xf5)[0x7ff9390ed615]
[79d169cd1242:09792] [ 4] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(+0x13cb3)[0x7ff939955cb3]
[79d169cd1242:09792] *** End of error message ***
WARNING: Logging before flag parsing goes to stderr.
  2019-07-28 00:23:36,255 - __main__ - initialize_data - DEBUG - Initialized Features: ['Date', 'Open', 'High', 'Low', 'Close', 'Volume']
[I 2019-07-28 00:24:12,288] Using an existing study with name 'PPO2__MlpLnLstmPolicy__IncrementalProfit' instead of creating a new one.
  2019-07-28 00:24:12,292 - __main__ - initialize_optuna - DEBUG - Initialized Optuna:
  2019-07-28 00:24:13,278 - __main__ - initialize_optuna - DEBUG - Best reward in (554) trials: -57.909999999999854
  2019-07-28 00:24:13,407 - __main__ - __init__ - DEBUG - Initialize RLTrader: PPO2__MlpLnLstmPolicy__IncrementalProfit
[79d169cd1242:09990] *** Process received signal ***
[79d169cd1242:09990] Signal: Segmentation fault (11)
[79d169cd1242:09990] Signal code: Address not mapped (1)
[79d169cd1242:09990] Failing at address: 0x7fe6bc31520d
[79d169cd1242:09990] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x12890)[0x7fe6befbf890]
[79d169cd1242:09990] [ 1] /lib/x86_64-linux-gnu/libc.so.6(getenv+0xa5)[0x7fe6bebfe785]
[79d169cd1242:09990] [ 2] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(_ZN13TCMallocGuardD1Ev+0x34)[0x7fe6bf469e44]
[79d169cd1242:09990] [ 3] /lib/x86_64-linux-gnu/libc.so.6(__cxa_finalize+0xf5)[0x7fe6bebff615]
[79d169cd1242:09990] [ 4] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(+0x13cb3)[0x7fe6bf467cb3]
[79d169cd1242:09990] *** End of error message ***

_Originally posted by @cyberslam in https://github.com/notadamking/RLTrader/issues/129#issuecomment-515723529_
